portfolio/src/main/webapp/index.html

```diff
@@ -13,44 +13,42 @@
       }
     </style>
   </head>
-  <body onload="fetchBlobstoreUrlAndShowForm()">
+  <body onload="loadPosts()">
     <button class="tablink" onclick="openPage('Blog', this, 'grey')" id="defaultOpen">Blog</button>
     <button class="tablink" onclick="openPage('Information', this, 'grey')" >Information</button>
     <button class="tablink" onclick="openPage('Resources', this, 'grey')">Resources</button>
-
     
- 
     <div id="Blog" class="tabcontent">
       <button onclick="showPostForm()">Click here to create a post</button>
       <button onclick="setNicknameForm()">Set your nickname</button>
       <div id="blogcontent">
         <h1>Blog</h1>
         <p>Welcome to the BIPOC blog, a space to uplift BIPOC and share resources.</p>
-        
-          <form id="my-form" enctype="multipart/form-data" method="POST">
-            <p>Type your post here:</p>
-            <textarea name="sender">Your name</textarea>
-            <br/>
-            <textarea name="text-input">Your text here</textarea>
-            <br/><br/>
-            <p>Upload an image:</p>
-            <input type="file" name="image">
-            <br/><br/>
-            <textarea name="tags">Tags</textarea>
-            <br/>
-            <p>Number of Posts to Display:</p>
-            <input type="number" name="comments-choice" id="comments-choice" min="1" max="100" value="1">
-            <br/><br/>
-            <textarea name="replies">Your reply here</textarea>
-            <br/>
-            <input type="submit"/>
-          </form>
-          <button onclick="getComments()">Show Comments</button>
- 
-        <div id="comments-container"></div>
-        <form action="/delete-data" method="POST">
-          <button onclick="deleteData()">Delete Comments</button>
+        <form id="my-form" enctype="multipart/form-data">
+          <textarea id="sender" rows="1" cols="20" placeholder="Nickname"></textarea> 
+          <br>
+          <textarea id="text-input" rows="6" cols="60" placeholder="What's on your mind..."></textarea>
+          <br>
+          <label for="tags">Choose tags:</label>
+          <select name="tags" id="tags">
+            <option value="#general">Default</option>
```

Comment:
Hmm, why would we hard code these tags on the client side? I thought they should be something sent from the server side on loading the pages.

Reply:
Actually, never mind.